src/Cache/Cache.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\ApiClient\Cache;
+
+class Cache implements CacheInterface
```

Comment:
1. why not using symfony cache component? Here it's a in memory cache, which is quite limited. It could be filesystem, redis, etc...

2. I think LRUCache (or similar) implementation would be better in term of DX, to avoid to fill the whole memory. 

3. Also, I'm wondering at which point it makes sense to not use HTTP protocol for caching (which means handling cache on PIM side). It would benefit all Akeneo API client. Did yo consider this option?

Reply:
Caching is a huge topic. This is only a first implementation and we will iterate on it to improve global performance sync.
For sure, having a real memory cache implementation would be better and we also need to handle cache invalidation later. We are going forward step by step ;)